model/patch_lifecycle.go

```diff
@@ -452,6 +452,14 @@ func FinalizePatch(ctx context.Context, p *patch.Patch, requester string, github
 		return nil, errors.Wrap(err, "fetching patch parameters")
 	}
 
+	authorEmail := p.GitInfo.Email
```

Comment:
The patch's Author field and the corresponding version's Author/AuthorID correspond to the id of the user who created the patch. 
Could it work to fetch the user with that instead?

Reply:
That's a good idea since it may work for more kinds of patches

Reply:
Good point sure!